Plugins/Events/Events/ObjectEvents.cpp

```diff
@@ -117,6 +132,43 @@ int32_t AddUseObjectActionHook(CNWSObject *thisPtr, ObjectID oidObjectToUse)
     return retVal;
 }
 
+void OpenInventoryHook(CNWSPlaceable *thisPtr, ObjectID oidOpener)
+{
+
+    auto PushAndSignal = [&](const std::string& ev) -> bool {
+        PushEventData("OBJECT", Utils::ObjectIDToString(thisPtr->m_idSelf));
+        return SignalEvent(ev, oidOpener);
+    };
```

Comment:
I'd prefer SignalEvent being called with `thisPtr->m_idSelf` so `OBJECT_SELF` is the placeable, since that's most, if not all events do, and push the opener as event data, eg: `PushEventData("OPENER", Utils::ObjectIDToString(oidOpener));`

Same for the CloseInventory event